core/templates/dev/head/pages/exploration_player/PlayerConstants.js

```diff
@@ -22,6 +22,19 @@ oppia.constant('TWO_CARD_THRESHOLD_PX', 960);
 
 oppia.constant('CONTINUE_BUTTON_FOCUS_LABEL', 'continueButton');
 
+// Wait period between successive hints.
+oppia.constant('WAIT_FOR_HINT_MSEC', 30000);
+
+// Delay for a hint to appear on clicking the hint button.
+oppia.constant('DELAY_FOR_HINT_FEEDBACK_MSEC', 600);
```

Comment:
The time delay between the learner clicking the hint button and the appearance of the hint.